lib/content-services/src/lib/content-node-share/content-node-share.dialog.spec.ts

```diff
@@ -28,9 +28,10 @@ import { RenditionService } from '../common/services/rendition.service';
 
 import { SharedLinksApiService } from './services/shared-links-api.service';
 import { ShareDialogComponent } from './content-node-share.dialog';
-import moment from 'moment';
 import { ContentTestingModule } from '../testing/content.testing.module';
 import { TranslateModule } from '@ngx-translate/core';
+import { format } from 'date-fns';
+import endOfDay from 'date-fns/endOfDay';
```

Comment:
First of all you're missing curly braces for this import and I think it should be combined with the upper one like `{ endOfDay, format }`